src/components/MyKiva/BorrowerCarousel.vue

```diff
@@ -58,7 +58,7 @@
 					ref="carousel"
 					class="borrower-carousel tw-w-full tw-overflow-visible"
 					:multiple-slides-visible="true"
-					:slide-max-width="singleSlideWidth"
+					:slide-max-width="'336px'"
```

Comment:
This probably isn't new to this changeset, but I'm seeing this error currently locally when clicking the carousel arrows. Maybe it's a quick fix?

```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'tabContext')
    at onInteractCarousel (BorrowerCarousel.vue:251:13)
    at callWithErrorHandling (chunk-TJ4MTCWI.js?v=f1e4d7fa:2487:19)
    at callWithAsyncErrorHandling (chunk-TJ4MTCWI.js?v=f1e4d7fa:2494:17)
    at emit (chunk-TJ4MTCWI.js?v=f1e4d7fa:7657:5)
    at chunk-TJ4MTCWI.js?v=f1e4d7fa:9185:45
    at chunk-7HKXJEJN.js?v=f1e4d7fa:2457:9
```